### PR TITLE
Add mpOpentracing-1.0 and change opentracing to support the TCK

### PR DESCRIPTION
**Pull request by kgibm**

### CODE REVIEW

**Comment by kgibm** [comment]
#build

**Comment by kgibm** [comment]
#build

File: dev/com.ibm.websphere.appserver.mpOpentracing-1.0/resources/l10n/com.ibm.websphere.appserver.mpOpentracing-1.0.properties [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+# -------------------------------------------------------------------------------------------------
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=The MicroProfile Opentracing specification provides a mechanism to access an opentracing.io Tracer
```
Comment (bcpswope) [inline dev/com.ibm.websphere.appserver.mpOpentracing-1.0/resources/l10n/com.ibm.websphere.appserver.mpOpentracing-1.0.properties]: I looked at the properties files entry:
The MicroProfile Opentracing specification provides a mechanism to access an opentracing.io Tracer (.)

Just looks like it is missing a period at the end of the sentence.

**Comment by kgibm** [comment]
#build

**Comment by kgibm** [comment]
#build

**Comment by kgibm** [comment]
#build

File: dev/com.ibm.ws.opentracing.cdi/resources/META-INF/beans.xml [hunk 2]
```diff
@@ -14,7 +14,7 @@
    version="1.1" 
    bean-discovery-mode="all">
    <scan>
-       <exclude name="com.ibm.ws.opentracing.cdi.**" />
```
Comment (Emily-Jiang) [inline dev/com.ibm.ws.opentracing.cdi/resources/META-INF/beans.xml]: you should directly remove the scan elements.

File: dev/com.ibm.ws.opentracing.cdi/src/com/ibm/ws/opentracing/cdi/TracedInterceptor.java [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+
+    /**
+     * See class comment.
+     *
+     * @param context Information about the wrapped method.
+     * @return Result of the invoking the method.
+     * @throws Exception Thrown by wrapped method.
+     */
+    @AroundInvoke
+    public Object executeFT(InvocationContext context) throws Exception {
```
Comment (Emily-Jiang) [inline dev/com.ibm.ws.opentracing.cdi/src/com/ibm/ws/opentracing/cdi/TracedInterceptor.java]: executeFT should be just execute. I think this is copy and paste error from Fault Tolerance:o.

**Comment by kgibm** [comment]
#build

**Comment by kgibm** [comment]
@bynumb1 or @Steve-Fontes Failures in com.ibm.websphere.appserver.osgiBundle_fat and com.ibm.ws.app.manager_fat seem to be unrelated, so this is ready for a final review.

**Pull request merged**